tutorials/translation/ct2_hindi_translation.ipynb

```diff
@@ -0,0 +1,623 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    from IndicTransToolkit import IndicProcessor\n",
+    "except ImportError:\n",
+    "    raise ImportError(\n",
+    "        \"IndicTransToolkit not found. Please install it using the following command: \\n\"\n",
+    "        + \"pip install git+https://github.com/VarunGumma/IndicTransToolkit.git\"\n",
+    "    )\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Table of content\n",
+    "1. [Ctranslate2 model integration](#ctranslate2-model-integration).\n",
+    "2. [Define IndicTranslation class](#define-indictranslation-class).\n",
+    "3. [Start the dask cluster](#start-the-dask-cluster).\n",
+    "4. [Define input](#define-input).\n",
+    "5. [Define output directory](#define-output-directory).\n",
+    "6. [Start the trasnaltion](#start-the-translation)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true
+   },
+   "source": [
+    "### CTranslate2 Model Integration\n",
+    "\n",
+    "We'll now create a custom CTranslate2 model class, which is essential for performing inference on the CT2 converted model. This example uses ctransalte2 model from [here](https://indictrans2-public.objectstore.e2enetworks.net/it2_preprint_ckpts/en-indic-preprint.zip), taken from IndicTrans2 github repo, [here](https://github.com/AI4Bharat/IndicTrans2/tree/main?tab=readme-ov-file#multilingual-translation-models). CTranslate2 is a C++ and Python library for efficient inference with Transformer models. You can read more about it [here](https://github.com/OpenNMT/CTranslate2). One of the features of it is, it enables fast and efficient execution on both CPU and GPU.\n",
+    "\n",
+    "This class will be integrated with CrossFit's `HFModel` class to leverage CrossFit's efficient batching and processing capabilities.\n",
+    "\n",
+    "#### Key Components:\n",
+    "\n",
+    "1. **CT2CustomModel**: A custom class for CTranslate2 model inference.\n",
+    "2. **ModelForSeq2SeqModel**: An extension of CrossFit's `HFModel` class, tailored for our translation task.\n",
+    "3. **TranslationConfig**: A dataclass for managing translation-specific configuration parameters.\n",
+    "\n",
+    "These model definitions are inspired by examples from the CrossFit project. For reference, you can find similar implementations in the [CrossFit GitHub repository](https://github.com/rapidsai/crossfit/pull/83/files#diff-d3c29a7456aac8be2bb3d53ba3d983e36631ea8dd36c4e52d9f3217183d4568f).\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "@dataclass\n",
```

Comment:
How about removing this entire cell and creating a new script in https://github.com/NVIDIA/NeMo-Curator/tree/main/nemo_curator/classifiers?

Reply:
Ah ok I see, using a utils file sounds good to me.